README.md

```diff
@@ -94,7 +94,9 @@ You can browse a list of articles that review or use the SG-NEx data [here](/doc
 
 The following short tutorials are available that demonstrate how to analyse the SG-NEx data:
 
-- [Transcript discovery and quantification of SG-NEx samples (using Bambu)](./docs/SG-NEx_Bambu_tutorial.md)
+- [Transcript discovery and quantification of SG-NEx samples (using Bambu)](./docs/bambu_tutorial.ipynb) <a target="_blank" href="https://colab.research.google.com/github/GoekeLab/sg-nex-data/blob/master/docs/SG-NEx_Bambu_tutorial_notebook.ipynb">
```

Comment:
I'm not sure what is better, to keep the original version that uses a mix of shell commands and R commands (easy to copy and follow on your own machine) plus the new google colab notebook (using the links to google colab), or to only keep the google colab notebook which will be easier to maintain?